lib/internal/options.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
-const { getOptions, shouldNotRegisterESMLoader } = internalBinding('options');
+const {
+  getOptions,
+  shouldNotRegisterESMLoader,
+  noGlobalSearchPaths
```

Comment:
nit: ASCII order + trailing comma
```suggestion
  noGlobalSearchPaths,
  shouldNotRegisterESMLoader,
```